src/main/index.js

```diff
@@ -320,16 +321,6 @@ const init = $context => {
 
   SideElement.init()
 
-  // Google Analytics opt out
-  $('a[href=ga-opt-out]').click(function(e) {
-    e.preventDefault()
-    gaOptout()
-    SystemNotification.notify(
-      t('Successfully deactivated Google Analytics'),
```

Comment:
is it possible to still show this notification, when analytics gets deactivated?